Introduction to Coroutines and Channels/06_StructuredConcurrency.md

```diff
@@ -32,39 +32,39 @@ fun main() = runBlocking { /* this: CoroutineScope */
 }
 ```
 
-When we call `launch` inside `runBlocking`, we call it as an extension to the implicit receiver
+If we were to call `launch` inside `runBlocking`, we would call it as an extension to the implicit receiver
 of the `CoroutineScope` type.
 Alternatively, we could explicitly write `this.launch`.
 
-We can say that the nested coroutine (started by `launch` in this example) is a child of the outer coroutine
+We could say that the nested coroutine (started by `launch` in this example) is a child of the outer coroutine
 (started by `runBlocking`). This "parent-child" relationship works through scopes: the child coroutine is started from
 the scope corresponding to the parent coroutine.
 
 It is possible to create a new scope without starting a new coroutine.
 The `coroutineScope` function does this. 
-When you need to start new coroutines in a structured way inside a `suspend` function without access to the outer scope,
+If we ever needed to start new coroutines in a structured way inside a `suspend` function without access to the outer scope,
```

Comment:
Similar: we do that (we start new coroutines in a structured way inside a suspend function...) right below, I don't understand 'if we ever needed' here.